src/test/java/org/folio/bulkops/service/OpenCSVConverterTest.java

```diff
@@ -8,58 +8,7 @@
 import org.folio.bulkops.BaseTest;
-import org.folio.bulkops.domain.bean.AddressType;
-import org.folio.bulkops.domain.bean.AddressTypeCollection;
-import org.folio.bulkops.domain.bean.BulkOperationsEntity;
-import org.folio.bulkops.domain.bean.CallNumberType;
-import org.folio.bulkops.domain.bean.CallNumberTypeCollection;
-import org.folio.bulkops.domain.bean.CirculationNote;
-import org.folio.bulkops.domain.bean.CustomField;
-import org.folio.bulkops.domain.bean.CustomFieldCollection;
-import org.folio.bulkops.domain.bean.CustomFieldTypes;
-import org.folio.bulkops.domain.bean.DamagedStatus;
-import org.folio.bulkops.domain.bean.DamagedStatusCollection;
-import org.folio.bulkops.domain.bean.Department;
-import org.folio.bulkops.domain.bean.DepartmentCollection;
-import org.folio.bulkops.domain.bean.ElectronicAccessRelationship;
-import org.folio.bulkops.domain.bean.ElectronicAccessRelationshipCollection;
-import org.folio.bulkops.domain.bean.Format;
-import org.folio.bulkops.domain.bean.HoldingsNoteType;
-import org.folio.bulkops.domain.bean.HoldingsNoteTypeCollection;
-import org.folio.bulkops.domain.bean.HoldingsRecord;
-import org.folio.bulkops.domain.bean.HoldingsRecordsSource;
-import org.folio.bulkops.domain.bean.HoldingsRecordsSourceCollection;
-import org.folio.bulkops.domain.bean.HoldingsType;
-import org.folio.bulkops.domain.bean.HoldingsTypeCollection;
-import org.folio.bulkops.domain.bean.Instance;
-import org.folio.bulkops.domain.bean.InstanceFormat;
-import org.folio.bulkops.domain.bean.InstanceFormats;
-import org.folio.bulkops.domain.bean.InstanceStatus;
-import org.folio.bulkops.domain.bean.InstanceStatuses;
-import org.folio.bulkops.domain.bean.InstanceType;
-import org.folio.bulkops.domain.bean.InstanceTypes;
-import org.folio.bulkops.domain.bean.Item;
-import org.folio.bulkops.domain.bean.ItemLocation;
-import org.folio.bulkops.domain.bean.ItemLocationCollection;
-import org.folio.bulkops.domain.bean.LoanType;
-import org.folio.bulkops.domain.bean.LoanTypeCollection;
-import org.folio.bulkops.domain.bean.MaterialType;
-import org.folio.bulkops.domain.bean.MaterialTypeCollection;
-import org.folio.bulkops.domain.bean.ModeOfIssuance;
-import org.folio.bulkops.domain.bean.ModesOfIssuance;
-import org.folio.bulkops.domain.bean.NatureOfContentTerm;
-import org.folio.bulkops.domain.bean.NatureOfContentTerms;
-import org.folio.bulkops.domain.bean.NoteType;
-import org.folio.bulkops.domain.bean.NoteTypeCollection;
-import org.folio.bulkops.domain.bean.SelectField;
-import org.folio.bulkops.domain.bean.SelectFieldOption;
-import org.folio.bulkops.domain.bean.SelectFieldOptions;
-import org.folio.bulkops.domain.bean.StatisticalCode;
-import org.folio.bulkops.domain.bean.StatisticalCodeCollection;
-import org.folio.bulkops.domain.bean.TextField;
-import org.folio.bulkops.domain.bean.User;
-import org.folio.bulkops.domain.bean.UserGroup;
-import org.folio.bulkops.domain.bean.UserGroupCollection;
+import org.folio.bulkops.domain.bean.*;
```

Comment:
@AntonAntonich it is necessary to import all needed classes

Reply:
done